homeassistant/components/sensor/icons.json

```diff
@@ -3,6 +3,9 @@
     "_": {
       "default": "mdi:eye"
     },
+    "area": {
+      "default": "mdi:flash"
```

Comment:
Seems like an incorrect icon for area.

Reply:
Oops that was temporary, will correct that

Reply:
picked square, seems the closest thing, otherwise thought of using mdi:all to keep it similar to distance.